networking/external_dns_operator/nw-external-dns-operator-configuration-parameters.adoc

```diff
@@ -0,0 +1,80 @@
+[id="nw-external-dns-operator-configuration-parameters_{context}"]
+= External DNS Operator configuration parameters
+
+[cols="3a,8a",options="header"]
+|===
+|Parameter |Description
+
+|`spec`
+|`spec` parameter enables
+
+[source,yaml]
+----
+spec:
+  provider:
+    type: AWS <1>
+    aws:
+      credentials:
+        name: aws-access-key <2>
+----
+<1>
+<2>
+
+|`zones`
+|`zones` enables your specified AWS zone. When you do not specify zones, ExternalDNS discovers all the zones in the account.
+
+|`domains`
+|`domains` parameter manages hostnames.
+
+[source,yaml]
+----
+domains:
+- filterType: Include <1>
+  matchType: Exact <2>
+  name: "myzonedomain1.com" <3>
+- filterType: Include
+  matchType: Pattern <4>
+  pattern: ".*\\.otherzonedomain.com" <5>
+----
+<1>
+<2>
+<3>
+<4>
+<5>
+
+|`source`
+|`source`
+
+[source,yaml]
+----
+source:
+  type: Service <1>
+  service:
+    serviceType:
+      - LoadBalancer <2>
+      - ClusterIP <3>
+  labelFilter:
+    matchLabels:
+      external-dns.mydomain.org/publish: "yes" <4>
+  hostnameAnnotation: "Allow" <5>
```

Comment:
Allow hostname annotation when generating DNS names. @arjunrn  can you provide more details on hostname annotation as you have worked on this ?
Default value is Ignore and when it is Ignore one needs to set fqdnTemplate.

Reply:
@arjunrn can you please check the above ?

Reply:
https://github.com/openshift/openshift-docs/pull/40441/files#r797553673

Reply:
ack